### ISSUE TITLE
Scripts to Finetune EfficientNet

### ISSUE THREAD

**Issue by D-X-Y** [facebookresearch/FixRes#20]
Hi authors, thanks for this great repo and interesting work.
I'm trying to reproduce your results of fine-tuning EfficientNet on ImageNet. Would you mind providing a fine-tuning script for EfficientNet?

**Comment by KoalaSheep** [comment]
I'm also interested in the EfficientNet fine-tuning script, @TouvronHugo can you send me a fine-tuning script for EfficientNet?

**Comment by D-X-Y** [comment]
From my experiments, no need to fine-tune, just pick up the pre-trained efficientnet model and test on a larger resolution and you will get better results

**Comment by KoalaSheep** [comment]
Thank you for your information. Since pre-trained FixEfficientNet weight is not provided, did you achieve the same results as reported with noisy student efficientnet weight, even without fine-tuning? I also notice that author added label smoothing in data-augmentation compared with other backbones.

**Comment by TouvronHugo** [comment]
Hi,
Increasing the resolution at test time will improve the results, fine-tuning will improve them more as shown in the FixRes article.
For fine-tuning the EfficientNet you can use EfficientNet model from the GitHub repo:
https://github.com/rwightman/pytorch-image-models
and use the current script to fine-tune them.
I will try to rapidly incorporate the efficientNet into the current script.
Best,
Hugo

**Comment by D-X-Y** [comment]
@TouvronHugo thanks for your reply.
Would you mind letting me know the improvement of fine-tuned vs pre-trained  efficientnet models when use larger test resolution?

**Comment by TouvronHugo** [comment]
@D-X-Y yes of course, 
For EfficientNet it's about ~0.1%-0.2%  if I'm not mistaken. For the ResNet and ResNeXt it is ~0.6%. 
I think it is possible to do better with EfficientNet by further optimising the fine-tuning parameters.

**Comment by D-X-Y** [comment]
@TouvronHugo Thanks for your reply and this useful information.  Yeap, we are trying to use AutoHAS (https://arxiv.org/pdf/2006.03656.pdf) to optimize such fine-tuning parameters.
